Server/Components/Pawn/utils.hpp

```diff
@@ -849,8 +849,30 @@ inline bool GetCurrentWorkingDirectory(std::string& result)
 	}
 	return ret;
 }
-
 #endif
+inline void NormalizeScriptName(std::string name, std::string& result)
+{
+#ifdef WIN32
```

Comment:
since open.mp can be compiled as 64 bit binary..
it is preferred to be:
```c++
#if defined(_WIN32) || defined(WIN32) || defined(__WIN32__) || defined(_WIN64) || defined(WIN64) || defined(__WIN64__)
```

Reply:
Thank you!